src/lib/udmp-parser.h

```diff
@@ -930,13 +1003,9 @@ class UserDumpParser {
     return true;
   }
 
-  const std::map<uint64_t, MemBlock_t> GetMem() const { return Mem_; }
-
-  const MemBlock_t *GetMemBlock(const void *Address) const {
-    return GetMemBlock(uint64_t(Address));
-  }
+  const std::map<uint64_t, MemBlock_t> &GetMem() const { return Mem_; }
 
-  const MemBlock_t *GetMemBlock(const uint64_t Address) const {
+  const MemBlock_t *GetMemBlock(const uintptr_t Address) const {
```

Comment:
I think `uint64_t` here is required; because in theory you can open a 64-bit dump from a 32-bit build; at least that was the goal behind it.